packages/tokens/src/semantic-tokens.ts

```diff
@@ -55,7 +55,7 @@ export const semanticTokens = {
         value: { base: '{colors.lightModeBrown.4}', _dark: '{colors.darkModeBrown.5}' },
       },
       'component-background-hover': {
-        value: { base: '#9486771A', _dark: '#9486771A' },
+        value: { base: 'rgba(148, 134, 119, 0.1)', _dark: 'rgba(148, 134, 119, 0.1)' },
```

Comment:
I know we had `hex` here before but we need to add this as a proper colour to `colors.ts` then include that colour here.

Is this coming from some design? Can we fund out from them what the name of it should be? 

We should add them like the others in `colors`:
```
  lightModeStacks: { value: '#5546FF' },
  darkModeStacks: { value: '#7F80FF' },
  overlay: { value: 'rgba(0,0,0,0.4)' },
```

Reply:
hm, yeah, agree here
https://github.com/leather-wallet/extension/issues/4805 link to issue 🤔

Reply:
We had a call about this and I will fix this. I'll:
- name `rgba(148, 134, 119, 0.1)` as `componentBackgroundHover` in `colors` 
- get rid of the unused `ink` tokens in there
- merge this and publish the new colours package